### PR TITLE
Normalize job properties

### PR DESCRIPTION
**Pull request by fabyan-alexander**

### CODE REVIEW

**Comment by fabyan-alexander** [comment]
[[BillingAg] Study how properties works on Spark](https://trello.com/c/UMu0y1D6/966-billingag-study-how-properties-works-on-spark)

File: src/test/java/com/leaf/billing/PropertyParserTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @Test
+    void decodeSourceDeleteFiles() {
+        String key = PropertyBuilder.SOURCE_DELETE_FILES;
+        Boolean value = true;
+        String[] args = {key + "=" + value};
+        JobProperties jobProperties = PropertyBuilder.build(args);
+        assertThat(jobProperties).hasFieldOrProperty(key);
+        assertThat(jobProperties).hasFieldOrPropertyWithValue(key, value);
+    }
+
```
Comment (ricardosantacattarina) [inline src/test/java/com/leaf/billing/PropertyParserTest.java:38]: from what I could understand, the PropertyBuilder should be able to handle multiple arguments at the same time, why don't we have a test where we test both source and target paths as args?

**Review by ricardosantacattarina — APPROVED** [review]
Nice!

File: src/test/java/com/leaf/billing/PropertyParserTest.java (hunk 1 above)
Comment (fabyan-alexander) [reply src/test/java/com/leaf/billing/PropertyParserTest.java:38]: I create a test to test each property, but merging then into a single test is also possible :sweat_smile:

**Review by DouglasVinicius — APPROVED** [review]
Nice work! :D
All looks good.

**Pull request merged**